engine/src/player-platform.cpp

```diff
@@ -996,8 +998,9 @@ void MCPlayer::close()
         m_is_attached = false;
     }
     // PM-2014-11-03: [[ Bug 13917 ]] m_platform_player should be recreated when reopening a recently closed stack, to take into account if the value of dontuseqt has changed in the meanwhile
+    // PM-2015-03-13: [[ Bug 14821 ]] Use a bool to decide whether to recreate a player, since assigning nil to m_platform_player caused player to become unresponsive when switching between cards
     if (m_platform_player != nil)
-        m_platform_player = nil;
```

Comment:
Does this not cause a memory leak? If m_platform_player is not nil, and we assign nil then we lose the pointer to the player object without freeing it.